src/session/types.ts

```diff
@@ -27,3 +27,7 @@ export interface ISignInInfo {
   signedIn?: boolean;
   userProfile?: IUserProfile;
 }
+
+export interface IPromiseToFileSession {
+  company_number: string;
```

Comment:
Again, we've got that COMPANY_NUMBER constant in keys.ts. Should it be used/referenced here or should it perhaps be deleted?

Reply:
It should be renamed.